tests/commands/test_cli.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from click.testing import CliRunner
+from tutor.commands.cli import *
+
+
+class CliTests(unittest.TestCase):
+    def test_help(self) -> None:
+        runner = CliRunner()
+        result = runner.invoke(print_help)
+        self.assertEqual(0, result.exit_code)
+        self.assertEqual(None, result.exception)
```

Comment:
This call and all similar lines should be converted to `self.assertIsNone(result.exception)`.